alyssabull/idk
# Feature/save delete activities

## What does this PR do?
- Adds `SavedActivities.js` and `ActivityCard.js` components
- Allows users to save and remove from saved on a button click
- Allows viewers to view all their saved activities and remove them from that view
- Refactors `RandomActivity.js` component to utilize react hooks for effectively
- Saves activities into localStorage to persist upon page refresh

## Where should the reviewer start?
- `SavedActivities.js` and `ActivityCard.js`

## How should this manually be tested?
- Save an activity, view on the saved activities page, delete the activity
- Save several activities, refresh the page to see them persist on refresh
